__tests__/users/App.test.js

```diff
@@ -1,7 +1,7 @@
 const puppeteer = require('puppeteer');
 const { filteredUsersData } = require('../../mock-data/users');
 const { mockUserData } = require('../../mock-data/users/mockdata');
-const API_BASE_URL = 'https://staging-api.realdevsquad.com';
```

Comment:
Why did you. change this?

Reply:
Testing runs on localhost:8000 and in the loadENV file we have declared this
```javascript
window.API_BASE_URL = 'https://api.realdevsquad.com';

if (
  window.location.hostname !== 'dashboard.realdevsquad.com' &&
  window.location.hostname !== 'localhost'
) {
  window.API_BASE_URL = 'https://staging-api.realdevsquad.com';
}
``` 
- Since the condition for switching to the staging API (https://staging-api.realdevsquad.com) only applies if the hostname is not dashboard.realdevsquad.com or localhost, testing on localhost will default to the production API (https://api.realdevsquad.com).
- After inspecting the API calls made during testing, you can confirm that no staging API requests are being sent. To provide controlled data responses for specific requests, you’ll need to adjust the base URL in the intercept function. This allows you to mock API responses directly within the test file when needed.